user_guide/src/SUMMARY.md

```diff
@@ -0,0 +1,81 @@
+# Summary
```

Comment:
This file is not in use anymore

Reply:
Cheers, these files were just left over on my branch as they were open in VScode, I'll rm them